src/genome/gene/mutationRateGene.h

```diff
@@ -0,0 +1,40 @@
+//
+// Created by Rosa on 1/12/2023.
+//
+
+#ifndef ALIFE_MUTATIONRATEGENE_H
+#define ALIFE_MUTATIONRATEGENE_H
+
+#include <initializer_list>
+#include "../gene.h"
+
+class MutationRateGene : public Gene {
+protected:
+	auto writeBody(Genome & genome) -> void override;
+
+public:
+	enum MutationType {
+		SUBSTITUTION = 0,
```

Comment:
I might have missed it, but what do each of these mutations do? I don't see anything in the .cpp file that maybe explains it more.